lib/interface_quda.cpp

```diff
@@ -5043,6 +5043,7 @@ void updateGaugeFieldQuda(void* gauge,
    if (param->use_resident_gauge) {
      if (!gaugePrecise) errorQuda("No resident gauge field to use");
      cudaGauge = gaugePrecise;
+     gaugePrecise = nullptr;
```

Comment:
What's the purpose of this addition? More specifically, why wasn't it there in the first place?

Reply:
Later 5070 bad things might happen if gaugePrecise and cudaGauge alias (I think)